src/samples/EducationStart/index.tsx

```diff
@@ -267,12 +276,17 @@ const EducationStartCase: FunctionComponent<any> = () => {
     return (
       <AppContextEducation.Provider
         value={{
-          appBacklinkProps: {},
+          appBacklinkProps: {
+            appBacklinkAction: () => {
+              setShowPega(false);
+              setShowLandingPage(true);
+            },
+            appBacklinkText: 'BACK'
+          },
```

Comment:
AppContextEducation.Provider is not required here as this is already there in useStartMashup. Please check once.

Reply:
updated. Also, the backlink text if translated before passing to context doesn't work so keeping the current approach unchanged.